src/components/HomeSection.tsx

```diff
@@ -40,14 +40,17 @@ export const HomeSection = ({
   title,
   titleDetail
 }: Props) => {
-  const {
-    data,
-    loading: isLoading,
-    refetch
-  } = useQuery(getQuery(query), {
-    variables: queryVariables,
-    fetchPolicy
-  });
+  const { data, isLoading, refetch } = useQuery(
+    [query, queryVariables],
+    async () => {
+      const client = await ReactQueryClient();
+
+      return await client.request(getQuery(query), queryVariables);
+    },
+    {
+      keepPreviousData: true
```

Comment:
i will also remove this here, see https://github.com/smart-village-solutions/smart-village-app-app/pull/1205#discussion_r1808830666